src/utils/fetcher.ts

```diff
@@ -10,6 +10,24 @@ const SIG_HEADER_NAME = "x-guild-sig"
 const PARAMS_HEADER_NAME = "x-guild-params"
 const AUTH_FLAG_HEADER_NAME = "x-guild-auth-location"
 
+const v2Replacer = (endpoint: string, options: Record<string, any>) => {
+  // if (options.method?.toLowerCase() === "post" && endpoint.includes("/v1/guild")) {
+  //   return endpoint.replace("/v1/guild", "/v2/guild/with-roles")
+  // }
+  // if (options.method?.toLowerCase() === "get" && endpoint.includes("/v1/guild/")) {
+  //   return endpoint.replace("/v1/guild/", "/v2/guild/guild-page/")
+  // }
+
+  if (
+    (options.method?.toLowerCase() ?? "get") === "get" &&
```

Comment:
Do we actually need this line? 👀 I guess we won't post/put/delete to a `/guild?` endpoint anyways, and "get" === "get" will always be true? But I might miss something here haha

Reply:
Oh, right, since the `?` is there, checking the method doesn't seem necessary 👍 
The `?? "get"` is just there, because for GETs, we don't specify it explicitly, and it is just `undefined`. So `undefined` means `"get"` here

Reply:
Removed in https://github.com/agoraxyz/guild.xyz/pull/803/commits/eb301d4a52c0170c87384e4677c59880cca3b045 👍